src/api/spec/models/architecture_spec.rb

```diff
@@ -3,4 +3,11 @@
 RSpec.describe Architecture do
   it { is_expected.to validate_presence_of(:name) }
   it { is_expected.to validate_uniqueness_of(:name) }
+
+  describe '#worker' do
+    let(:i586) { Architecture.find_by(name: 'i586') }
+
+    it { expect(i586).to respond_to(:worker) }
```

Comment:
This line is pointless with the line below. I would remove it

Reply:
yup, TDD, I will remove it